lib/OpenQA/WebAPI/Controller/API/V1/JobTemplate.pm

```diff
@@ -442,14 +442,7 @@ sub create {
         $status = 400;
     }
     else {
-        if (defined($affected_rows)) {
-            $json->{affected_rows} = $affected_rows;
-            $self->emit_event('openqa_jobtemplate_create', {affected_rows => $affected_rows});
-        }
-        else {
-            $json->{id} = $id;
-            $self->emit_event('openqa_jobtemplate_create', {id => $id});
-        }
+        $self->emit_event('openqa_jobtemplate_create', {id => $_}) for @ids;
```

Comment:
I'd prefer

```
$self->emit_event('openqa_jobtemplate_create', {ids => \@ids});
```

so it is clear which job templates have been created together and we create less events.

Reply:
How would you link them? My assumption in #3329 is that each event is one item. The same is true of search results, should we add events there.

Reply:
Mh… ok, I suppose for the sake of it might be easier to have individual events. But it would also be possible to render the list of IDs like "1, 2, 3, 4" where each ID is an individual link.

Reply:
I'm also updating the description... but I decided to disambiguate the `id` issue by introducing a `job_group_id` and `ids` for individual job templates, regardless of how the event is emitted. And it's always one event per API call.